github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -1881,6 +1881,35 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: true
+    annotations:
+      fork-per-release: "true"
+    cluster: prow-workloads
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 1h0m0s
+    max_concurrency: 11
+    name: pull-kubevirt-code-lint
+    optional: true
+    skip_branches:
+    - release-\d+\.\d+
+    skip_report: false
+    spec:
+      containers:
+      - command:
+        - /usr/local/bin/runner.sh
+        - /bin/sh
+        - -c
+        - |
+          make lint
+        image: quay.io/kubevirtci/bootstrap:v20220512-78048f1
+        name: ""
+        resources:
+          requests:
+            memory: 4Gi
+      nodeSelector:
+        type: ibm-prow-jobs
```

Comment:
@rhrazdil This one is redundant because you will need to define `cluster: ibm-prow-jobs`

Reply:
Right, got it, thanks.
[Removed.](https://github.com/kubevirt/project-infra/compare/5dae944164a57c5c72d9a42c9769cb7fb1b2cb25..fbabfb67e773c6b1d7750ebaec719c181c69afbd)